doc/quickstart/grass_quickstart.rst

```diff
@@ -346,13 +336,10 @@ available. For advanced use `Pythons Ctypes` is supported allowing the
 Python programmer direct access to GRASS GIS' extensive C libraries.
 See the manual pages for an extensive description of the programming options.
 
-Shutdown and the command line
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When finished, exit the GRASS GUI with :menuselection:`File --> Exit GUI`.
-Before you close the GRASS terminal session as well, try a GRASS
-module by typing "``g.manual --help``" which will give you a list
-of module options. The GRASS command line is where the true power of
+Command line
+--------------------------------------------------------------------------------
+In the GRASS terminal session, try a GRASS module by typing "``g.manual --help``" which will give you a list of module options. 
```

Comment:
```suggestion
In the GRASS terminal session, try a GRASS GIS module by typing "``v.clean --help``"
which will give you a list of module options. 
```
(changed to v.clean since g.manual is not really a GIS related command)

Reply:
done.